app/src/screens/mapscreen.js

```diff
@@ -226,10 +232,19 @@ function MapScreen(): React.Node {
         layers={layers}
         getTooltip={(p: Point) => p.message}>
         <ReactMapGL mapboxApiAccessToken={MAPBOX_TOKEN} mapStyle={mapStyle}>
-          <div style={{position: 'absolute', right: 0}}>
+          <div style={{ position: 'absolute', right: 0 }}>
             <NavigationControl showCompass={true} showZoom={false} />
           </div>
         </ReactMapGL>
+        <div className={classes.root}>
+          <AppBar position="static">
+            <Toolbar>
+              <Typography variant="h6" className={classes.title}>
+                3D Coverage Maps
+              </Typography>
+            </Toolbar>
+          </AppBar>
+        </div>
```

Comment:
Is this after the ReactMapGL for any specific reason? 
(And if so, maybe we should add a comment about that)

Reply:
The ReactMapGL layer extends under the entire viewscape.  I don't think it matters where it is, but various code snippets on the Deck.gl and react-maps-gl documentation usually have it at the top.